### PR TITLE
Update checkout-test-plots.include.yml to cache@v3 and run ./build-workflows.py

### PR DESCRIPTION
**Pull request by aqk**

### CODE REVIEW

File: tests/runner_templates/checkout-test-plots.include.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@
       run: date +%F > today.txt
 
     - name: Cache test blocks and plots
-      uses: actions/cache@v2
```
Comment (aqk) [inline tests/runner_templates/checkout-test-plots.include.yml:6]: @AmineKhaldi the jobs run this [pre-commit.yml](https://github.com/Chia-Network/chia-blockchain/blob/main/.github/workflows/pre-commit.yml), but then they check to see if the workflow files match after running [build-workflows.py](https://github.com/Chia-Network/chia-blockchain/blob/main/tests/build-workflows.py)

File: .github/workflows/build-test-macos-plot_sync.yml [hunk 2]
```diff
@@ -65,13 +65,25 @@ jobs:
         restore-keys: |
           ${{ runner.os }}-pip-
 
-    - name: Checkout test blocks and plots
-      uses: actions/checkout@v3
+    - name: Daily (POC) cache key invalidation for test blocks and plots
```
Comment (aqk) [inline .github/workflows/build-test-macos-plot_sync.yml:68]: @AmineKhaldi Your new code was not being included here.

Comment (altendky) [reply .github/workflows/build-test-macos-plot_sync.yml:68]: This was handled in https://github.com/Chia-Network/chia-blockchain/pull/11092.

**Comment by altendky** [comment]
I think this is closable with the reworked test workflows.

**Comment by emlowe** [comment]
Closing old PR

**Pull request closed without merging**